ToolTip.ios.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
-var React = require('react-native');
 var {
   requireNativeComponent,
   TouchableHighlight,
   View,
-} = React;
-
-var ToolTipMenu = React.NativeModules.ToolTipMenu;
-
+  NativeModules,
+} = require('react-native');
+var React = require('react');
+var findNodeHandle = require('react-native/Libraries/ReactNative/findNodeHandle');
```

Comment:
`findNodeHandle` is it not exported from react-native?

var { NativeModules, findNodeHandle } = require('react-native');